components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/resources/userstore.yaml

```diff
@@ -315,6 +315,42 @@ paths:
           $ref: '#/components/responses/ServerError'
         '501':
           $ref: '#/components/responses/NotImplemented'
+  '/userstores/meta/types/{type-id}/attributes':
+    get:
+      tags:
+        - Meta
+      summary: >-
+        Retrieve the meta attributes of a user store of a given user store type.
+      operationId: getUserStoreAttributeMappings
+      description: >
+        This API provides the capability to retrieve the attribute mappings
+        of a given user store type
+
+         <b>Permission required:</b>
+
+        */permission/admin
```

Comment:
We need to mention the specific permission and the scope. The current level says we need to have all the admin permissions. Shall we fix this?